src/Analysis/Ast/Impl/Documents/RunningDocumentTable.cs

```diff
@@ -65,12 +66,14 @@ private class DocumentEntry {
             lock (_lock) {
                 entry = FindDocument(null, uri);
                 if (entry == null) {
+                    var resolver = _services.GetService<IPythonInterpreter>()?.ModuleResolution?.CurrentPathResolver;
```

Comment:
What is the case when `_services.GetService<IPythonInterpreter>()` or `ModuleResolution` is null?